eng/tools/versioning/set-dev.js

```diff
@@ -48,7 +48,7 @@ const commitChanges = async (rushPackages, package) => {
 const updatePackageVersion = (rushPackages, package, buildId) => {
   const currentVersion = rushPackages[package].json.version;
   const parsedVersion = semver.parse(currentVersion);
-  rushPackages[package].newVer = `${parsedVersion.major}.${parsedVersion.minor}.${parsedVersion.patch}-dev.${buildId}`;
+  rushPackages[package].newVer = `${parsedVersion.major}.${parsedVersion.minor}.${parsedVersion.patch}-alpha.${buildId}`;
```

Comment:
@weshaggard is `buildId` here still a date? is this what we want?

Reply:
Yes see the guidance at https://azure.github.io/azure-sdk/policies_releases.html#javascript. The alpha builds will still be date based.